functions/Find-DbaAgentJob.ps1

```diff
@@ -0,0 +1,137 @@
+{
+<#
+.SYNOPSIS 
+Find-DbaAgentJob finds agent job/s that fit certain search filters.
+
+.DESCRIPTION
+This command filters SQL Agent jobs giving the DBA a list of jobs that may need attention or could possibly be options for removal. 
+	
+.PARAMETER SqlServer
+The SQL Server instance. You must have sysadmin access and server version must be SQL Server version 2000 or higher.
+
+.PARAMETER SqlCredential
+Allows you to login to servers using SQL Logins as opposed to Windows Auth/Integrated/Trusted. 
+
+.PARAMETER LastUsed
+Find all jobs that havent ran in the INT number of previous day(s)
+
+.PARAMETER Disabled
+Find all jobs that are disabled
+	
+.PARAMETER NoSchedule
+Find all jobs with schedule set to it
+	
+.PARAMETER EmailNotification
+Find all jobs without email notification configured
+
+.PARAMETER Filter
+Allows you to enter an array of agent job names to ignore 
+
+.PARAMETER Detailed
+Returns a more detailed output showing why each job has been reported
+	
+.NOTES 
+Author: Stephen Bennett: https://sqlnotesfromtheunderground.wordpress.com/
+Requires: sysadmin access on SQL Servers
+dbatools PowerShell module (https://dbatools.io, clemaire@gmail.com)
+
+Copyright (C) 2016 Chrissy LeMaire
+This program is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.
+This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+You should have received a copy of the GNU General Public License along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+.LINK
+https://dbatools.io/Find-DbaAgentJob
+
+.EXAMPLE
+Find-DbaAgentJob -SQLServer Dev01 -LastUsed 10 
+Returns all agent job(s) that have not ran in 10 days
+
+.EXAMPLE 
+Find-DBAAgentJob -SQLServer Dev01 -Disabled -EmailNotification -NoSchedule -Detailed
+Returns all agent job(s) that are either disabled, have no email notification or dont have a schedule. returned with detail
+
+.EXAMPLE
+Find-DbaAgentJob -SQLServer Dev01 -LastUsed 10 -Filter "Yearly - RollUp Workload", "SMS - Notification" 
+Returns all agent jobs that havent ran in the last 10 ignoring jobs "Yearly - RollUp Workload" and "SMS - Notification" 
+#>
+	[CmdletBinding()]
+	Param (
+		[parameter(Position = 0, Mandatory = $true, ValueFromPipeline = $True)]
+		[Alias("ServerInstance", "SqlInstance", "SqlServers")]
+		[string[]]$SqlServer,
+		[System.Management.Automation.PSCredential]$SqlCredential,
+		[int]$LastUsed,
+        [switch]$Disabled,
+        [switch]$NoSchedule,
+        [switch]$EmailNotification,
+        [string[]]$Filter,
+        [switch]$Detailed
+	)
+	BEGIN
+	    {
+            $output = @()
+	    }
+	PROCESS
+	    {
+		    FOREACH ($instance in $SqlServer)
```

Comment:
Actually we need both. A function like Get-DbaPowerplan works on a computer (host), while this one needs SQL instance(s). Therefore I find it important to use both where they're appropriate. Since 'server' is the most ambiguous, I prefer $computername for computers and $instance(name) for instances.